### ISSUE TITLE
More Compatible Teacher Pred

### ISSUE THREAD

**Issue by didadida-r** [fschmid56/EfficientAT#6]
Hi, 
i try to run the training script, and some data was loss during downloading. So the teacher pred index is not right and can not be used.
I am wondering if your have time to update a Dict version of teacher pred with audio_name as key, because it would be more complex for new guy to get the ensemble logit

```
# here the teacher_preds is indexed by dataset index
if args.kd_lambda > 0:
      y_soft_teacher = teacher_preds[i]
```

Thanks

**Comment by fschmid56** [comment]
Thanks for the request. This is already the third request regarding this. I'll put it on top of my list for next week to add the dict.

**Comment by fschmid56** [comment]
I've uploaded the file [fname_to_index](https://github.com/fschmid56/EfficientAT/releases/download/v0.0.1/fname_to_index.pkl), which contains a dict converting the file IDs to the indices in the [predictions file](https://github.com/fschmid56/EfficientAT/releases/download/v0.0.1/passt_enemble_logits_mAP_495.npy). 
Tell me if this has the right format for you.

**Comment by fschmid56** [comment]
As pointed out in [this issue](https://github.com/fschmid56/EfficientAT/issues/7), there has been a problem with the _fname_to_index_ file. You can resolve this issue by re-downloading the file if you have already done so.
